### ISSUE TITLE
Adding an OCSP Validator to a certificate object with a "chainCA" property does not work as expected.

### ISSUE THREAD

**Issue by ycgemalto** [F5Networks/f5-appsvcs-extension#749]
### Environment
 * Application Services Version: 3.46.0
 * BIG-IP Version: 15.1.2.1

### Summary
Please also include information about the reproducibility and the severity/impact of the issue.

I am using AS3 to add a OCSP validator to an certificate object but I get an Err. 422 back when I sent the request.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "class": "AS3",
    "action": "deploy",
    "persist": true,
    "logLevel": "debug",
    "declaration": {
        "class": "ADC",
        "schemaVersion": "3.19.0",
        "id": "hod-partitions-api-1",
        "label": "hsmod-crypto-api",
        "remark": "Crypto API for DpoD",
        "AS3_HSMOD_test": {
            "class": "Tenant",
            "HSMOD": {
                "class": "Application",
                "template": "generic",
                "na_hsm_dpod_live_cert": {
                        "class": "Certificate",
                        "certificate": "-----BEGIN CERTIFICATE-----\n...\n-----END CERTIFICATE-----\n",
                        "privateKey": "-----BEGIN RSA PRIVATE KEY-----\n...\n-----END RSA PRIVATE KEY-----\n",
                        "chainCA": "-----BEGIN CERTIFICATE-----\n...\n-----END CERTIFICATE-----\n",
                        "staplerOCSP": {
                            "use": "na_hsm_dpod_live_ocsp"
                        },
                        "issuerCertificate": {
                            "use": "na_hsm_dpod_live_ocsp_signing_cert"
                        }
                },
                "na_hsm_dpod_live_ocsp": {
                        "class": "Certificate_Validator_OCSP",
                        "dnsResolver": {
                            "bigip": "/Common/root resolver"
                        }
                },
                "na_hsm_dpod_live_ocsp_signing_cert": {
                        "class": "Certificate",
                        "certificate": "-----BEGIN CERTIFICATE-----\n...\n-----END CERTIFICATE-----\n"
                }
            }
        }
    }
}
```

2. Observe the following error response:
```json
    {
      "code": 422,
      "message": "declaration failed",
      "response": "01071bb1:3: The certificate (/AS3_HSMOD_test/HSMOD/na_hsm_dpod_live_cert-bundle.crt) is self-signed, so can not have an issuer certificate configured.",
      "host": "localhost",
      "tenant": "AS3_HSMOD_test",
      "runTime": 1497
    }
```

### Expected Behavior
The validator should be added to the certificate object "na_hsm_dpod_live_cert" and not "na_hsm_dpod_live_cert-bundle" as the error seems to indicate.

### Actual Behavior
A clear and concise description of what actually happens.
Please include any applicable error output.

See error above.

I have also tried not having the "chainCA" property in the certificate object when I added the OCSP validator and then PATCH the property back in the certificate object with this:

```json
[
    {
        "op": "add",
        "path": "/AS3_HSMOD_test/HSMOD/na_hsm_dpod_live_cert/chainCA",
        "value": "-----BEGIN CERTIFICATE-----\n...\n-----END CERTIFICATE-----\n"
    }
]
```
but ended up with this error:
```json
    {
      "code": 422,
      "message": "declaration failed",
      "response": "01071bb1:3: The certificate (/AS3_HSMOD_test/HSMOD/na_hsm_dpod_live_cert-bundle.crt) is self-signed, so can not have an issuer certificate configured.",
      "host": "localhost",
      "tenant": "AS3_HSMOD_test",
      "runTime": 2342
    }
```
So again, the PATCH was being done on the wrong object!

**Comment by sunitharonan** [comment]
Thanks @ycgemalto , in order to prioritize and understand the issue better, please reach out to us at automation_toolchain_pm@f5.com

**Comment by sunitharonan** [comment]
Closing as this has been released in AS3 3.48.0
